src/Components/Login/Login.tsx

```diff
@@ -19,56 +13,61 @@ const ALL_USERS = gql `
   }
 `
 
-const ALL_BOOKS = gql `
-  query AllBooks {
-    books {
-      id
-      imageUrl
-    }
-  }
-`
-
 const Login = () => {
   const [email, setEmail] = useState('')
   const [username, setUsername] = useState('')
   const [userLocation, setUserLocation] = useState('')
   const [activeAccount, setAccount] = useState(true)
-  const [allUsers, setUsers] = useState([{userName: "Josh", emailAddress: "test", location: "Denver"}])
+  const [loginError, setLoginError] = useState(false)
+  const [errorMessage, setErrorMessage] = useState('')
+  const [allUsers, setUsers] = useState([{emailAddress: "joshua@gmail.com", userName: "Joshua", location: "Colorado"}])
 
   let navigate = useNavigate()
 
-  // const { data, loading, error } = useQuery(ALL_USERS)
-  const { data, loading, error } = useQuery(ALL_BOOKS)
-
-  console.log("Endpoint", error)
+  const { data, loading, error } = useQuery(ALL_USERS)
   
-
   const verifyLogin = () => {
      let noUser = allUsers.map(user => {
       if(user.emailAddress !== email){
-        setAccount(false)
+        setLoginError(true)
+        setErrorMessage("Could not find that email please try again")
+        setEmail('')
       } else {
         setAccount(true)
         navigate("/home")      
       }
-      
-     })
-     return noUser
+    }) 
+      return noUser
   }
 
   const createUser = () => {
-    let newUser = {userName: username, emailAddress: email, location: userLocation}
-
-    console.log(newUser);
-  
+    if(username === '') {
+      setLoginError(true)
+      setErrorMessage("Please fill in a username")
+    }
+    if(email === '') {
+      setLoginError(true)
+      setErrorMessage("Please fill in an email")
+    }
+    if(userLocation === '') {
+      setLoginError(true)
+      setErrorMessage("Please fill in a location")
+      console.log("Email: ", email, "Username: ", username);
+      
+    } else {
+      let newUser = {userName: username, emailAddress: email, location: userLocation}
+      setEmail('')
+      setUsername('')
+      setUserLocation('')
+      navigate("/home")
+    } 
   }
 
   return (
     <section className='login-page'>
-      <h1>Welcome to Book Worm</h1>
       <div className='login-container'>
-        <h2>Login/ Create Account</h2>
-        {activeAccount === false && <p>Could not find account please create a new one</p>}
+      <h1 className='page-title'>Book Worm</h1>
+      {loginError === true && <h3>{errorMessage}</h3>}
```

Comment:
So when it comes to the <h#> tags I always like to make sure we use the right ones for the right reasons. Apologies if this is preachy.

When we use the header tags, it should always be for categorical hierarchy of information on the page. Not for font-sizes. Meaning, `<h1>` should always be the most important content on the page. `<h2>` is for content belonging to `<h1>`, `<h3>` for content belonging to `<h2>` etc.

You can have multiple `<h1>`'s on a page, this simply means that both of these "sections" are of equal importance and you should dive down the first one, then proceed to the next one.

This is important because this is exactly how screen-readers look at these tags. Screen readers will determine what parts of the page are organized in what hierarchy by using these.

So when I see an `<h1>`, followed by an `<h3>` it doesn't make sense. You're missing your `<h2>`.

As for font sizes, all of that can be controlled via CSS.
I know this is a bit of a rant, so please let me know if you have any questions on this and I'd be happy to go over it.